scripts/configs/interactive_learning.yaml

```diff
@@ -0,0 +1,65 @@
+# Interactive learning experiments.
+---
+APPROACHES:
+  main:
+    NAME: "interactive_learning"
+    FLAGS:
+      explorer: "greedy_lookahead"
+  # MLP baseline
+  main-mlp:
+    NAME: "interactive_learning"
+    FLAGS:
+      explorer: "greedy_lookahead"
+      interactive_num_ensemble_members: 1
+  # query baselines
+  ask-all:  # section kid
+    NAME: "interactive_learning"
+    FLAGS:
+      explorer: "greedy_lookahead"
+      interactive_query_policy: "nonstrict_best_seen"
+      interactive_score_function: "trivial"
+  ask-none:  # silent kid
+    NAME: "interactive_learning"
+    FLAGS:
+      explorer: "greedy_lookahead"
+      interactive_query_policy: "threshold"
+      interactive_score_function: "trivial"
+      interactive_score_threshold: 1.0
+  ask-random:  # random kid
+    NAME: "interactive_learning"
+    FLAGS:
+      explorer: "greedy_lookahead"
+      interactive_query_policy: "random"
+      interactive_score_function: "trivial"
+      interactive_random_query_prob: 0.03  # queries about the same amount as main
+  # action baselines
+  glib:
+    NAME: "interactive_learning"
+    FLAGS:
+      explorer: "glib"
+  random-actions:
+    NAME: "interactive_learning"
+    FLAGS:
+      explorer: "random_options"
+  no-actions:
+    NAME: "interactive_learning"
+    FLAGS:
+      explorer: "no_explore"
+ENVS:
+  cover:
+    NAME: "cover"
+    FLAGS: {}
+ARGS: {}
+FLAGS:  # common args
+  excluded_predicates: "Covers,Holding"
+  interactive_score_threshold: 0.05
```

Comment:
I see that you removed `--interactive_score_function entropy` because it's the default in `settings.py` which makes sense. why don't we similarly change the defaults for the other key parameters here in settings.py, rather than overriding them here, now that we're converging on a final set of hyperparameters. so:
* `interactive_score_threshold`
* `predicate_mlp_classifier_max_itr`

I guess that's it